lib/sorbet-rails/model_plugins/enumerable_collections.rb

```diff
@@ -1,71 +0,0 @@
-# typed: strict
-require ('sorbet-rails/model_plugins/base')
-class SorbetRails::ModelPlugins::EnumerableCollections < SorbetRails::ModelPlugins::Base
-
-  sig { override.params(root: Parlour::RbiGenerator::Namespace).void }
-  def generate(root)
-    # model relation, assocation relation, & association proxy are enumerable
-    # we need to implement "each" in these methods so that they work
-    model_relation_class_rbi = root.create_class(self.model_relation_class_name)
-    create_enumerable_methods_for(model_relation_class_rbi, include_methods: false)
-
-    model_assoc_relation_rbi = root.create_class(self.model_assoc_relation_class_name)
-    create_enumerable_methods_for(model_assoc_relation_rbi)
-
-    model_assoc_proxy_class_rbi = root.create_class(self.model_assoc_proxy_class_name)
-    create_enumerable_methods_for(model_assoc_proxy_class_rbi)
-
-    # following methods only exists in an association proxy
-    ["<<", "append", "push", "concat"].each do |method_name|
-      elem = self.model_class_name
-      model_assoc_proxy_class_rbi.create_method(
-        method_name,
-        parameters: [
-          Parameter.new("*records", type: "T.any(#{elem}, T::Array[#{elem}])"),
-        ],
-        return_type: "T.self_type",
-      )
-    end
-  end
-
-  sig { params(class_rbi: Parlour::RbiGenerator::ClassNamespace, include_methods: T::Boolean).void }
-  def create_enumerable_methods_for(class_rbi, include_methods: true)
-    class_rbi.create_include("Enumerable")
-
-    # An escape hatch that prevents these methods from being added unnecessarily.
-    # In certain cases, we can remove these from the classes because they can
-    # be generalized to use Elem in a superclass instead.
-    if include_methods
-      class_rbi.create_method(
-        "each",
-        parameters: [
-          Parameter.new("&block", type: "T.proc.params(e: #{self.model_class_name}).void")
-        ],
-        return_type: "T::Array[#{self.model_class_name}]",
-        implementation: true,
-      )
-      class_rbi.create_method(
```

Comment:
I think this method (flatten) is missing from the new generation code. Can you add it?

Reply:
I don't think `flatten` is a method on ActiveRecord::Relation or related classes:
```ruby
> User.all.flatten
NoMethodError: undefined method `flatten' for #<User::ActiveRecord_Relation:0x00007f8c7eb904d0>
```

Reply:
But if it's part of an array, it works 🤷 
```
> [User.all].flatten
```